packages/config/config/devices/0x0002/hrvccm.json

```diff
@@ -1,4 +1,4 @@
-// Danfoss HRVCCM
+// Danfoss H
```

Comment:
wonder what happened here 🤷🏻‍♂️

Reply:
Looks like I just missed updating the comment when either merging or making the identifier more descriptive. Fixed.